### ISSUE TITLE
Monitor mode

### ISSUE THREAD

**Issue by shortwavedave** [redbear/IoT_pHAT#29]
Anyone know if this board can be used in monitor mode?

**Comment by smnfz** [comment]
Yeah! It works somehow, you get a strange error message in Kali_ARM when you put it in monitor mode (see below), but it still works and create the wlan0mon interface, and youre able to put the card in monitor mode and send de-auth, to catch a handshake, and it both sends it and catch it correctly However...

Running  "aireplay-ng -9 wlan0mon" returns with "working" however it only returns 1 of 30 injections made (3%)  despite its only 10 meters from router, so i suppose its only supporting it partly, or the drivers somehow is not working properly

Latest firmware 005..
error message from airmon-ng below
------------------------
root@kali:~# airmon-ng start wlan0

Found 2 processes that could cause trouble.
Kill them using 'airmon-ng check kill' before putting
the card in monitor mode, they will interfere by changing channels
and sometimes putting the interface back in managed mode

  PID Name
  227 NetworkManager
  333 wpa_supplicant

PHY	Interface	Driver		Chipset

phy0	wlan0		brcmfmac	Broadcom 43430

		(mac80211 monitor mode vif enabled for [phy0]wlan0 on [phy0]wlan0mon)
command failed: Unknown error 524 (-524)
		(mac80211 station mode vif disabled for [phy0]wlan0)

root@kali:~# 
-------------------------------------------------------------

**Comment by steev** [comment]
I just replied to a similar thread in the Kali forums - the main thing holding out on it, is actually the firmware.  We do not patch/include a patched version of that firmware.  https://github.com/seemoo-lab/nexmon/tree/master/firmwares/bcm43438/7_45_41_26 *should* work with the iot_pHAT, if I read the output correctly, but I don't have the hardware to test (and looking, amazon has one, but for 40 dollars......) so if someone would be willing to try to compile that firmware file and replace the one on the rpi system and reboot and check... that would be helpful

**Comment by smnfz** [comment]
> I just replied to a similar thread in the Kali forums - the main thing holding out on it, is actually the firmware. We do not patch/include a patched version of that firmware. https://github.com/seemoo-lab/nexmon/tree/master/firmwares/bcm43438/7_45_41_26 _should_ work with the iot_pHAT, if I read the output correctly, but I don't have the hardware to test (and looking, amazon has one, but for 40 dollars......) so if someone would be willing to try to compile that firmware file and replace the one on the rpi system and reboot and check... that would be helpful

Yeah.. That was me asking. It seems that i have some issues to log in to Kali forums again. However I’m happy that someone who knows what they’re doing takes a shot at this, re4son seems to be interested tho. and i have never experimented with drivers on Kali before
